parlai/core/torch_agent.py

```diff
@@ -699,8 +714,8 @@ def build_dictionary(self):
         """
         d = self.dictionary_class()(self.opt)
         if self.opt.get('person_tokens'):
-            d[self.P1_TOKEN] = 999999999
-            d[self.P2_TOKEN] = 999999998
+            d[self.P1_TOKEN] = 999_999_999
```

Comment:
😮 what is this?

Reply:
python 3.6 feature; the _ is just a comment symbol for more readability.